tests/acceptance/features-test.js

```diff
@@ -13,6 +15,16 @@ module('Acceptance: Features', {
   }
 });
 
+test('features are defined in app config', function() {
+  ok(features.enabled('feature-from-config'), 'feature-from-config is enabled automatically');
```

Comment:
In an acceptance test, I think it would be preferable to test this through the DOM, like the other acceptance tests. Importing features here and directly checking it is a weird mix of unit with acceptance.

There is also a unit test for the initializer that could be used.